study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/addOrEditStudyNotification.jsp

```diff
@@ -473,7 +473,7 @@ For studies that are already launched, notifications get scheduled for delivery
       if (dt < serverDateTime()) {
         $('.timepicker').parent().addClass('has-error has-danger').find(
             '.help-block.with-errors').empty().append($("<ul><li> </li></ul>").attr("class","list-unstyled").text(
-            "Please select a time that has not already passed for the current date"));
+            "Please select a time that has not gone past yet"));
```

Comment:
This wording sounds incorrect. Please use "Please select a time in the future."

Reply:
Fixed review comment